blockchain/chainio_test.go

```diff
@@ -143,52 +133,48 @@ func TestBlockIndexSerialization(t *testing.T) {
 		name       string
 		entry      blockIndexEntry
 		serialized []byte
-	}{
-		{
-			name: "no votes, no revokes",
-			entry: blockIndexEntry{
-				header:         baseHeader,
-				status:         statusDataStored | statusValidated,
-				voteInfo:       nil,
-				ticketsVoted:   nil,
-				ticketsRevoked: nil,
-			},
-			serialized: hexToBytes("040000001f733757b25d804863dbcad599c931e15e3a3425e21a6716690100000000000066c664d887c6e2c0c132b3c5c8c82f9931470d6ecbc5ccc003755d7979bbf25edec2d752551f38a2bd4bb0d06ff9a1ed06f833a5aa8dc12bdc27759b056529020100313e16e64c0b0400030274a10000986f011aee686fbd010000000f4b02001f2c000037c4665904f85df58f01ed92645e0a6b11ee3b3c000000000000000000000000000000000000000004000000030000"),
-		},
-		{
-			name: "1 vote, no revokes",
-			entry: blockIndexEntry{
-				header:         baseHeader,
-				status:         statusDataStored | statusValidated,
-				voteInfo:       baseVoteInfo[:1],
-				ticketsVoted:   baseTicketsVoted[:1],
-				ticketsRevoked: nil,
-			},
-			serialized: hexToBytes("040000001f733757b25d804863dbcad599c931e15e3a3425e21a6716690100000000000066c664d887c6e2c0c132b3c5c8c82f9931470d6ecbc5ccc003755d7979bbf25edec2d752551f38a2bd4bb0d06ff9a1ed06f833a5aa8dc12bdc27759b056529020100313e16e64c0b0400030274a10000986f011aee686fbd010000000f4b02001f2c000037c4665904f85df58f01ed92645e0a6b11ee3b3c00000000000000000000000000000000000000000400000003012e9b0f3f37c74ba8e8a9211d38e9706106ec482a1422a880ea53475477a8628b040100"),
+	}{{
+		name: "no votes",
+		entry: blockIndexEntry{
+			header:   baseHeader,
+			status:   statusDataStored | statusValidated,
+			voteInfo: nil,
 		},
-		{
-			name: "no votes, 1 revoke",
-			entry: blockIndexEntry{
-				header:         baseHeader,
-				status:         statusDataStored | statusValidated,
-				voteInfo:       nil,
-				ticketsVoted:   nil,
-				ticketsRevoked: baseTicketsRevoked[:1],
-			},
-			serialized: hexToBytes("040000001f733757b25d804863dbcad599c931e15e3a3425e21a6716690100000000000066c664d887c6e2c0c132b3c5c8c82f9931470d6ecbc5ccc003755d7979bbf25edec2d752551f38a2bd4bb0d06ff9a1ed06f833a5aa8dc12bdc27759b056529020100313e16e64c0b0400030274a10000986f011aee686fbd010000000f4b02001f2c000037c4665904f85df58f01ed92645e0a6b11ee3b3c000000000000000000000000000000000000000004000000030001ebe895e6b1a9397b3ea756c4a5ff1d3bd678293d2980bc8e00a8fc8f1bf04681"),
+		serialized: hexToBytes("040000001f733757b25d804863dbcad599c931e15e3a3" +
+			"425e21a6716690100000000000066c664d887c6e2c0c132b3c5c8c82f9931470" +
+			"d6ecbc5ccc003755d7979bbf25edec2d752551f38a2bd4bb0d06ff9a1ed06f83" +
+			"3a5aa8dc12bdc27759b056529020100313e16e64c0b0400030274a10000986f0" +
+			"11aee686fbd010000000f4b02001f2c000037c4665904f85df58f01ed92645e0" +
+			"a6b11ee3b3c0000000000000000000000000000000000000000040000000300"),
+	}, {
+		name: "1 vote, no revokes",
```

Comment:
Since revokes are no longer being tracked by the index entry this should be named `1 vote` instead.

Reply:
Good catch.  I updated the other test names, but missed this one.  Updated.